kuma/core/management/commands/fixaccount.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+
+from allauth.account.models import EmailAddress
+from allauth.socialaccount.models import SocialAccount
+
+
+class Command(BaseCommand):
+    help = 'Reconnects an MDN account (Persona) to an email address'
+
+    def add_arguments(self, parser):
+        parser.add_argument('username', nargs=1,
+                            help='MDN account username.')
+        parser.add_argument('email', nargs=1,
+                            help='Email address to connect.')
+
+    def handle(self, *args, **options):
+        username = options['username'][0]
+        email = options['email'][0]
+
+        User = get_user_model()
+
+        try:
+            user = User.objects.get(username=username)
+
+        except User.DoesNotExist:
+            raise CommandError('User %s does not exist.' % username)
+
+        if user.email != email:
+            self.stdout.write('Fixing email address in auth_user record.')
+            user.email = email
+            user.save()
+
+        # Fix the django-allauth EmailAddress record
+        emailaddress = EmailAddress.objects.get(user=user)
```

Comment:
I think if they used `createsuperuser`, then we shouldn't be doing anything in this command since that's outside the scope. The code should handle that case and provide appropriate instructions. I'll fix that. Good catch!